integreat_cms/cms/urls/protected.py

```diff
@@ -6,51 +6,50 @@
 from django.urls import include, path
 
 from ..forms import (
+    EventTranslationForm,
     LanguageForm,
     LanguageTreeNodeForm,
     OfferTemplateForm,
     OrganizationForm,
-    RegionForm,
-    EventTranslationForm,
-    POITranslationForm,
     PageTranslationForm,
+    POITranslationForm,
+    RegionForm,
 )
 from ..models import (
+    POI,
```

Comment:
I understand that POI is on top of this list because it's all caps, but I think I would rather have POI where it was before behind Page and before POICategory. At least that's where I would search for it. But maybe I'm just being picky